default.conf.template

```diff
@@ -0,0 +1,97 @@
+# http://nginx.org/en/docs/http/ngx_http_headers_module.html
```

Comment:
I do not understand the purpose of the custom config, which is the same as we have in web-base.
Can you explain, what I am missing, please?

Reply:
I left this for now here until next version of web-base gets released with the fixes so that we can demo or have others test. If web-base gets released before this I will remove it

Reply:
now 0.0.12 is released so I removed it from here